src/Compiler/Symbols/FSharpDiagnostic.fs

```diff
@@ -78,6 +78,17 @@ module ExtendedData =
         /// Represents the URL format of the diagnostic
         member this.UrlFormat: string = urlFormat
 
+    /// Additional data for diagnostics about experimental attributes.
+    [<Class; Experimental("This FCS API is experimental and subject to change.")>]
+    type ExperimentalExtendedData
```

Comment:
Are non-nullable strings the correct signature?
Is there a particular reason for representing missing attribute with an empty string here?

(Is it e.g. what the attribute does when e.g. scanned via reflection at runtime ? )

Reply:
Will have to check

Reply:
So yeah at runtime the absence of information is represented as null to I think it make sense to make this values optional None aka null